recipes/poco/all/conanfile.py

```diff
@@ -111,6 +110,12 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
+        if not self.options.enable_xml:
```

Comment:
Any specific version of Poco?

Reply:
I see the same functionality in earliest 1.8.1 and in latest 1.10.1